designs/2022-community-eslint-org/README.md

```diff
@@ -133,53 +133,71 @@ I personally think that the acceptance criteria for a package to go under the
   [`eslint-plugin-flowtype`](https://github.com/gajus/eslint-plugin-flowtype)
   and/or [`eslint-plugin-jsdoc`](https://github.com/gajus/eslint-plugin-jsdoc)
   if they would feel it would be better maintained by the community instead of
-  putting all workload on their shoulders.
+  putting all workload on their shoulders. I can see the same happening for
+  [`eslint-plugin-security`](https://github.com/nodesecurity/eslint-plugin-security),
+  which is currently maintained by the
+  [@nodesecurity](https://github.com/nodesecurity) team.
+
+- It should agree to the code of conduct of the org
+- It should have an OSI-approved license
+- `eslintbot` should be added as an owner to the original npm package
 
 If a package is meeting these criteria, but we can't get into contact with the
 owner of the repo to transfer ownership over to the new org, we could
-temporarily fork the repo & publish the package as
-`@eslint-community-fork/<package name>` or so.
+temporarily fork the repo into the new org. However, the ultimate goal would
+always be to transfer the original repo to the new org.  
+If we had a temporary fork, we should create PRs to the original repo to bring
+it up to date with the fork before continuing with other tasks.
 
 ### What permissions should maintainers have?
 
-I think it would be best to have some sort of a "core" team (of at least 3(?)
-people) that's making the decisions in terms of acceptance to the repo, which
-would (in addition to ESLint TSC) all be admin of the org.
+I think it would be best to have some sort of a community core team (of at least
+3(?) people) that's making the decisions in terms of acceptance to the repo,
+which would (in addition to ESLint TSC) all be admins of the org.
+
+We can make a team (with sub-teams, if we want) for each package, where we give
+maintainer rights on the repo for that specific package, but keep admin rights
+to ESLint TSC & the community core team. This way we prevent that the repo would
+be destroyed from the org.
+
+We'll create a setup where we publish new versions of the package with
```

Comment:
If you really and to do this, then we need more details on how it would work.

Reply:
It would be nice if you could help me understand what @eslintbot is doing on the main repo & how it's set up there in order to write down how we could use it in the community org.

Reply:
I don’t think we’d want the same setup. @eslintbot is a GitHub account, and this would only need npm. 

My suggestion: packages can tag and create GitHub releases, and that would trigger an action/workflow (unsure of specifics) that would use the npm eslintbot account to publish the release.